go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/evanphx/json-patch v4.11.0+incompatible
 	github.com/fatih/structs v1.1.0
 	github.com/form3tech-oss/jwt-go v3.2.2+incompatible
+	github.com/fsnotify/fsnotify v1.4.9
```

Comment:
It would be great if you explain why you add fsnotify dependency here.

IMO, license header fix does not need to add any dependency.

Reply:
Unnecessary, I will be remove. thanks.